metrics_sdk/lib/opentelemetry/sdk/metrics/instrument/up_down_counter.rb

```diff
@@ -36,6 +34,12 @@ def add(amount, attributes: nil)
             OpenTelemetry.handle_error(exception: e)
```

Comment:
Same thing here - the call to `update` should change.

Reply:
updated 8b149a742fa97eca626ed3a06967d96a7ab93408